packages/gatsby-source-drupal/src/utils.js

```diff
@@ -1,11 +1,74 @@
 const _ = require(`lodash`)
+const axios = require('axios')
 const { nodeFromData, downloadFile, isFileNode } = require(`./normalize`)
 
 const backRefsNamesLookup = new WeakMap()
 const referencedNodesLookup = new WeakMap()
 
-const handleReferences = (node, { getNode, createNodeId }) => {
+const fetchLanguageConfig = async ({ translation, baseUrl, apiBase, basicAuth, headers, params }) => {
+  if (!translation) {
+    return {
+      defaultLanguage: 'und',
+      enabledLanguages: ['und'],
+      translatableEntities: [],
+    }
+  }
+
+  const [availableLanguagesResponse, translatableEntitiesResponse] = await Promise.all([
+    axios.get(`${baseUrl}/${apiBase}/configurable_language/configurable_language`, {
+      auth: basicAuth,
+      headers,
+      params,
+    }),
+    axios.get(`${baseUrl}/${apiBase}/language_content_settings/language_content_settings?filter[language_alterable]=true`, {
+      auth: basicAuth,
+      headers,
+      params,
+    }),
+  ])
+
+  const enabledLanguages = (
+    availableLanguagesResponse
+    .data
+    .data
+    .filter(
+      language => [
+        `und`,
+        `zxx`
+      ].indexOf(language.attributes.drupal_internal__id) === -1
+    )
+    .map(language => language.attributes.drupal_internal__id)
+  )
+
+  const defaultLanguage = enabledLanguages[0]
+
+  const translatableEntities = (
+    translatableEntitiesResponse
+    .data
+    .data
+    .map(entity => ({
+      type: entity.attributes.target_entity_type_id,
```

Comment:
Don't need "type" or "bundle" since we are using "id". Might be better to just create a list of type strings preformatted eg. `node--article`

Reply:
Just notice: in Drupal we can alter that, we do not need to use type--bundle naming.

Reply:
@kmajzlik Do you mind explaining that a bit more? I'm just proposing removing the extra properties here and changing the returned value from an object to a string matching the existing "id" property.